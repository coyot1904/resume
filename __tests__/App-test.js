/**
 * @format
 */

 import 'react-native';
 import React from 'react';
 import PImage from './../src/component/Image';
 
 // Note: test renderer must be required after react-native.
 import {render, cleanup} from 'react-native-testing-library';
 
 
 afterEach(cleanup);
 
 
 describe('<PImage />', () => {
   it('should match snapshot', () => {
     const rendered = render(<PImage source={require('../../assets/img/logom.png')} />).toJSON();
     
     expect(rendered).toMatchSnapshot();
   });
 });
 