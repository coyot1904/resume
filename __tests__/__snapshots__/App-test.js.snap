// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PImage /> should match snapshot 1`] = `
<Image
  source={
    Object {
      "default": "",
    }
  }
/>
`;
